### DISCUSSION TITLE
We are getting "There were issues affecting this run of Lighthouse" error when we generate the lighthouse report for the page.

### DISCUSSION THREAD

**Discussion by GopinathS89** [GoogleChrome/lighthouse#13064] [Q&A]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

its happening local as well as in testing environment

### What happened?

There were issues affecting this run of Lighthouse:

The page may not be loading as expected because your test URL (https://www.test.com/testpage/add) was redirected to https://www.test.com/testpage/login?code=ere434343&grant-id=123sdfs443erewr423er. Try testing the second URL directly.

### What did you expect?

Is there any possibility to disable the page reload in the lighthouse?

### How were you running Lighthouse?

Chrome DevTools

### Lighthouse Version

8.1.0

**Comment by brendankenny** [comment]
> message: The page may not be loading as expected because your test URL was redirected to (second URL). Try testing the second URL directly.

> Is there any possibility to disable the page reload in the lighthouse?

If the page redirects, Lighthouse will follow the redirect. Lighthouse loads the page in Chrome mostly normally, so it acts just like a load in the browser would.

For testing, you could consider sniffing the UA string (it ends with `Chrome-Lighthouse`) and not redirecting if that's present (or check for known IP addresses) or other typical ways of dealing with bots visiting a page.
